src/content/api/compilation-hooks.md

```diff
@@ -393,30 +429,69 @@ Parameters: `modules` `records`
 
 Store chunk info to the records.
 
-Parameters: `chunks` `records`
+- Callback Parameters: `chunks` `records`
+
+
+### `optimizeCodeGeneration`
+
+T> This hook will be available in v5.0.0
+
+A plugin can tap into this hook to optimize the generated code.
+
+- Callback Parameters: `modules`
+
+
+### `beforeModuleHash`
```

Comment:
This and few following hooks are missing callback parameters

Reply:
That's because they don't pass parameters to the callbacks. We can write `- callback parameters: none` but I don't like it much

Reply:
Great, just confirming!